sdk/core/azure-core-tracing-opentelemetry/CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 - Aligned OpenTelemetry tracing infrastructure with OpenTelemetry 1.17.0 conventions for use with Azure Monitor.
 
 ### Other Changes
-- Updated test collateral to reflect Azure Core tracing changes.
+
```

Comment:
Should we decide against that item removal, at least this newline between "Other Changes" should still be checked in.